### ISSUE TITLE
[Bug]: Giving warning in next.js 15

### ISSUE THREAD

**Issue by anishkumar127** [microsoft/fluentui#33388]
### Component

Other...

### Package version

^9.56.1

### React version

19.0.0-rc-66855b96-20241106

### Environment

```shell
Development
```

### Current Behavior

**warnings**
 WARN  Issues with peer dependencies found
.
├─┬ react-redux 9.1.2
│ ├── ✕ unmet peer react@^18.0: found 19.0.0-rc-66855b96-20241106
│ └─┬ use-sync-external-store 1.2.2
│   └── ✕ unmet peer react@"^16.8.0 || ^17.0.0 || ^18.0.0": found 19.0.0-rc-66855b96-20241106
├─┬ @reduxjs/toolkit 2.3.0
│ └── ✕ unmet peer react@"^16.9.0 || ^17.0.0 || ^18": found 19.0.0-rc-66855b96-20241106
└─┬ @fluentui/react-components 9.56.1
  └─┬ @fluentui/react-alert 9.0.0-beta.124
    └─┬ @fluentui/react-avatar 9.6.42
      └─┬ @fluentui/react-popover 9.9.24
        └─┬ @fluentui/react-context-selector 9.1.69
          └── ✕ unmet peer scheduler@">=0.19.0 <=0.23.0": found 0.25.0-rc-66855b96-20241106

Done in 11s

**errors in ui**

@fluentui/react-provider: There are conflicting ids in your DOM. Please make sure that you configured your application properly. 
 
 Configuration guide: https://aka.ms/fluentui-conflicting-ids

Call Stack
createUnhandledError
file:///C:/Users/AnishKumar/Music/helpdesknext/helpdesk/.next/static/chunks/74038_next_dist_client_0dd523._.js (744:19)
handleClientError
file:///C:/Users/AnishKumar/Music/helpdesknext/helpdesk/.next/static/chunks/74038_next_dist_client_0dd523._.js (905:98)

![Image](https://github.com/user-attachments/assets/b1b900a8-151c-418c-a088-90647cdce5ad)

### Expected Behavior

Should not give the warning and error

### Reproduction

https://stackblitz.com/edit/stackblitz-starters-t6chfr?description=The%20React%20framework%20for%20production&file=package.json,app%2FHome.tsx,app%2Fpage.tsx&title=Next.js%20Starter

### Steps to reproduce

Install next.js 15 version and install fluent ui 9 this "@fluentui/react-components": "^9.56.1", version
and can't provide reproduction link because stackblitz default now only support the next.js 13 version but im current version is next.js of 15

### Are you reporting an Accessibility issue?

None

### Suggested severity

Urgent - No workaround and Products/sites are affected

### Are you willing to submit a PR to fix?

yes

### Validations

- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [x] The provided reproduction is a minimal reproducible example of the bug.

**Comment by spmonahan** [comment]
I'm not familiar with Next, but it doesn't look like you've configured `IdPrefixProvider` as you mentioned in your bug. Can you confirm whether you've tried this?

We also have docs for configuring Fluent with Next 13+ and SSR here: https://react.fluentui.dev/?path=/docs/concepts-developer-server-side-rendering-next-js-appdir-setup--docs

**Comment by anishkumar127** [comment]
> I'm not familiar with Next, but it doesn't look like you've configured `IdPrefixProvider` as you mentioned in your bug. Can you confirm whether you've tried this?
> 
> We also have docs for configuring Fluent with Next 13+ and SSR here: https://react.fluentui.dev/?path=/docs/concepts-developer-server-side-rendering-next-js-appdir-setup--docs

yes i did it next.js appDir provider via wrap the layout.tsx and wrap Fluent ui provider wrap with IdPrefixProvider which example are given into the somewhere in the previous issues on github but still not working.

**Comment by spmonahan** [comment]
I don't see the provider in [`layout.tsx`](https://stackblitz.com/edit/stackblitz-starters-t6chfr?file=app%2Flayout.tsx)? Am I missing something?

>this issue appears in the next.js 15 with react 19 RC
Does the issue go away with React 18?

**Comment by anishkumar127** [comment]
> I don't see the provider in [`layout.tsx`](https://stackblitz.com/edit/stackblitz-starters-t6chfr?file=app%2Flayout.tsx)? Am I missing something?
> 
> > this issue appears in the next.js 15 with react 19 RC
> 
> Does the issue go away with React 18?

**yes this issue go away with React 18.**

**Comment by spmonahan** [comment]
We don't currently support React 19 but the team is working on that.

@tudorpopams  and @dmytrokirpa for visibility.

**Comment by dggsax** [comment]
Hi! I'm starting a new NextJS project (first time using FluentUI since v8) and running into a similar issue, here's my package.json which is relatively simple. Let me know if it would help to publish the repo for looking into this. Context: I'm new to NextJS so it may be something obvious I'm missing:

```json
{
  "private": true,
  "scripts": {
    "dev": "next",
    "build": "next build",
    "start": "next start"
  },
  "dependencies": {
    "@fluentui/react-components": "^9.58.2",
    "@reduxjs/toolkit": "^2.5.1",
    "fluentui-next-appdir-directive": "^0.2.0",
    "next": "latest",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-redux": "^9.1.0"
  },
  "devDependencies": {
    "@types/node": "20.2.5",
    "@types/react": "18.2.37",
    "@types/react-dom": "^18.2.18",
    "typescript": "^5.3.3"
  }
}

```

**Comment by dmytrokirpa** [comment]
Hey @dggsax, check out this Next.js 15 + React 18 example - https://stackblitz.com/github/dmytrokirpa/fluentui-next/tree/react-18


Just a heads up, React 19 isn't supported yet. It's on our roadmap, but no clear timeline for it yet.

**Comment by dggsax** [comment]
Thank you so much! I'll check this out and follow-up if there were any subtle things I had done wrong that are worth mentioning in the documentation.

**Comment by creage** [comment]
We have same false positive error in dev mode, when running Next.JS 14.2.28 + FluentUI 9.62.0.

We have single `.fui-FluentProvider` in entire app, but it still throws that error in console.
